Source/WebKit/DerivedSources.make

```diff
@@ -450,6 +450,7 @@ $(WEB_PREFERENCES_PATTERNS) : $(WTF_BUILD_SCRIPTS_DIR)/GeneratePreferences.rb $(
 SERIALIZATION_DESCRIPTION_FILES = \
 	NetworkProcess/NetworkProcessCreationParameters.serialization.in \
 	Shared/API/APIGeometry.serialization.in \
+	Shared/Cocoa/ArgumentCodersCocoa.serialization.in \
```

Comment:
Could this file be named WebCoreArgumentCodersCocoa.serialization.in?  I think we will later want to add one called ArgumentCodersCocoa.serialization.in for structures that aren't in WebCore.